frontend/src/styles/utilities.scss

```diff
@@ -125,6 +125,38 @@
     }
 }
 
+// fractional widths
+.w-1\/2 {
+    width: 50%;
+}
+.w-1\/3 {
+    width: 33.333333%;
+}
+.w-2\/3 {
+    width: 66.666667%;
+}
+.w-1\/4 {
+    width: 25%;
+}
+.w-3\/4 {
+    width: 75%;
+}
+.w-1\/5 {
+    width: 20%;
+}
+.w-2\/5 {
+    width: 40%;
+}
+.w-3\/5 {
+    width: 60%;
+}
+.w-4\/5 {
+    width: 80%;
+}
+.w-full {
+    width: 100%;
+}
```

Comment:
This should be done inside the `@each $kind in ('width', 'height')` loop (and `.w-full` is already in there)

Reply:
Where is '.w-full'? Can't find it 🤔

Reply:
I also added this in a different pr :D Look for what Michael mentioned `@each $kind in (...`